package.json

```diff
@@ -3,30 +3,38 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "body-parser": "~1.17.1",
+    "babel-cli": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-register": "^6.26.0",
+    "body-parser": "^1.18.2",
+    "chai": "^4.1.2",
     "color": "^2.0.1",
-    "cookie-parser": "~1.4.3",
-    "debug": "~2.6.3",
-    "eslint": "^4.4.1",
+    "cookie-parser": "^1.4.3",
+    "debug": "^3.1.0",
+    "eslint": "^4.11.0",
     "eslint-config-google": "^0.9.1",
-    "express": "~4.15.2",
-    "firebase-admin": "^5.4.1",
+    "express": "^4.16.2",
+    "firebase-admin": "^5.5.1",
     "flags": "^0.1.3",
-    "hbs": "~4.0.1",
-    "morgan": "~1.8.1",
+    "hbs": "^4.0.1",
+    "mocha": "^4.0.1",
+    "morgan": "^1.9.0",
+    "node-sass": "^4.7.2",
     "promise-map-series": "^0.2.3",
-    "serve-favicon": "~2.4.2"
-  },
-  "devDependencies": {
-    "babel-preset-env": "^1.6.0",
-    "babel-register": "^6.26.0",
-    "chai": "^4.1.2",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^7.0.0",
-    "gulp-eslint": "^4.0.0",
-    "gulp-mocha": "^4.3.1",
-    "gulp-sass": "^3.1.0",
-    "mocha": "^4.0.0",
+    "serve-favicon": "^2.4.5",
     "supertest": "^3.0.0"
+  },
+  "scripts": {
```

Comment:
Add in a script to replicate the `build:clientjs` from the old Gulp file.

Reply:
Done in latest commit.